pkgs/by-name/no/notary/package.nix

```
@@ -1,41 +1,24 @@
-{ lib, fetchFromGitHub, buildGoPackage, libtool }:
-
-buildGoPackage rec {
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+buildGoModule {
   pname = "notary";
-  version = "0.6.1";
-  gitcommit = "d6e1431f";
+  version = "0.6.1-unstable-2024-04-16";
 
   src = fetchFromGitHub {
     owner = "theupdateframework";
     repo = "notary";
-    rev = "v${version}";
-    sha256 = "1ak9dk6vjny5069hp3w36dbjawcnaq82l3i2qvf7mn7zfglbsnf9";
+    rev = "9d2b3b35929392c9945d976b8bdecbe2f53a299e";
+    sha256 = "sha256-u19BfTJwRWholK0b3BcgSmcMM9AR7OeXo64AOi87r0A=";
   };
 
-  patches = [ ./no-git-usage.patch ];
-
-  buildInputs = [ libtool ];
-  buildPhase = ''
-    runHook preBuild
-    cd go/src/github.com/theupdateframework/notary
-    SKIPENVCHECK=1 make client GITCOMMIT=${gitcommit}
-    runHook postBuild
-  '';
-
-  goPackagePath = "github.com/theupdateframework/notary";
-
-  installPhase = ''
-    runHook preInstall
-    install -D bin/notary $out/bin/notary
-    runHook postInstall
-  '';
+  vendorHash = null;
```

```suggestion
  vendorHash = null;
  
    tags = [
    "pkcs11"
  ];

  ldflags = [
    "-X github.com/theupdateframework/notary/version.NotaryVersion=${version}"
  ];
```

See https://github.com/notaryproject/notary/blob/master/Makefile#L19
https://github.com/notaryproject/notary/blob/master/Makefile#L15